packages/adminrouter/extra/src/.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+
+# Ignore the following errors:
```

Comment:
To get here I think we need to be at a point where we run unittests and code quality with the packages we build rather than the packages that setuptools/on-host tox decides to build in. So, I am very OK with other projects having their own settings, but they should still use the same special linter. See: https://jira.mesosphere.com/browse/DCOS-14305